# generated by patch-package 6.4.8 on 2022-05-05 10:42:34
#
# command:
#   npx patch-package ts-jsdoc
#
# declared package:
#   ts-jsdoc: 3.2.2
#
diff --git a/node_modules/ts-jsdoc/out/manager.js b/node_modules/ts-jsdoc/out/manager.js
index a4564ad..7b7ce2d 100644
--- a/node_modules/ts-jsdoc/out/manager.js
+++ b/node_modules/ts-jsdoc/out/manager.js
@@ -69,7 +69,8 @@ async function generateAndWrite(basePath, config, tsConfig) {
                 externalToModuleName.set(value, oldPath.substring(oldPath.indexOf(":") + 1, dotIndex));
                 return `external:${value}`;
             }
-            return oldPath;
+            const lastIndex = oldPath.lastIndexOf("node_modules")
+            return lastIndex < 0 ? oldPath : ('module:' + oldPath.substring(lastIndex + 13));
         };
         let result = "";
         const externalToModuleName = new Map();
